var/spack/repos/builtin/packages/httpress/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Httpress(MakefilePackage):
+    """High performance HTTP server stress & benchmark utility."""
+
+    homepage = "https://bitbucket.org/yarosla/httpress/wiki/Home"
+    hg       = "https://bitbucket.org/yarosla/httpress"
+
+    version('develop')
+    version('1.1.0', revision='1.1.0', preferred=True)
+
+    depends_on('mercurial', type='build')
+    depends_on('libev')
+    depends_on('gnutls')
+
+    def install(self, spec, prefix):
+        mkdirp(prefix.bin)
+        mkdirp(prefix.obj)
```

Comment:
These `mkdirp` shouldn't be needed, `install_tree` already handles that.

Reply:
OK, remove them in https://github.com/spack/spack/pull/15935/commits/4103fb2a1690c93de114928d0149848a17723503